docker/Dockerfile.debian

```diff
@@ -15,6 +16,6 @@ RUN apt-get update && apt-get install -y -q --no-install-recommends \
     libx11-dev \
     libxext-dev \
     libxrender-dev \
-    python \
+    $python_package \
```

Comment:
```suggestion
    $python \
```

Reply:
I considered this, but I found it ambiguous. `python` could refer to Python package, or Python binary etc. Are you sure you want me to rename the variable?

Reply:
I think we use `jpeg` the same way, and it's understood that it's the package.